src/services/schema-file-updater.js

```diff
@@ -163,14 +154,33 @@ class SchemaFileUpdater {
       actionFormatted.fields = actionFormatted.fields || [];
       actionFormatted.fields = actionFormatted.fields.map((field) =>
         SchemaFileUpdater.formatObject(field, serializerOptions.actions.fields.attributes));
+
+      actionFormatted.hooks = actionFormatted.hooks || {};
+      actionFormatted.hooks = SchemaFileUpdater
+        .formatObject(action.hooks, serializerOptions.actions.hooks.attributes);
+      actionFormatted.hooks.load = Boolean(action.hooks && (typeof action.hooks.load === 'function'));
+      actionFormatted.hooks.change = action.hooks
+        && action.hooks.change
+        && typeof action.hooks.change === 'object'
+        ? Object.keys(action.hooks.change)
+        : [];
+
```

Comment:
Should we create a method entitled  `SchemaFileUpdater.cleanActionHooks` as we can see for `SchemaFileUpdater.cleanSegments` ?

Reply:
Yes! Done.